WooCommerce/Classes/Authentication/Jetpack Setup/Site Credential Login/SiteCredentialLoginView.swift

```diff
@@ -107,7 +107,7 @@ struct SiteCredentialLoginView: View {
                                                                   text: $viewModel.username,
                                                                   isSecure: false,
                                                                   errorMessage: nil,
-                                                                  isFocused: focusedField == .username))
+                                                                 isFocused: focusedField == .username, autocapitalization: .none))
```

Comment:
Nitpick: Indentation needs to be updated. We can also move `autocapitalization` to a new line for consistency.

Reply:
It seems other fields were not aligned. I updated it here 5661c09d63b6d9ad450a3f089e39a7faeab2a9ec